exercises/practice/word-search/tests/word_search.cairo

```diff
@@ -0,0 +1,734 @@
+fn Should_locate_a_left_to_right_word_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 10, end_col: 7, end_row: 10, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_that_left_to_right_word_in_a_different_position_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 10, end_col: 7, end_row: 10, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_different_left_to_right_word_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "fortranftw",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["fortran"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 2 words');
+
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "fortran", start_col: 1, start_row: 7, end_col: 7, end_row: 7, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_multiple_words() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "fortranftw",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["fortran", "clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 2, 'Expected to find 2 words');
+
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "fortran", start_col: 1, start_row: 7, end_col: 7, end_row: 7, }
+    );
+
+    assert_eq!(
+        results[1],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 10, end_col: 7, end_row: 10, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_single_word_written_right_to_left() {
+    #[cairofmt::skip]
+    let grid = array![
```

Comment:
The grid doesn't match [canonical data](https://github.com/exercism/problem-specifications/blob/main/exercises/word-search/canonical-data.json#L323). This is the 3rd mismatched test case, **please double-check them all**.

Reply:
ooops, sorry about this, made the changes and did cross check others.